float64 avr_error
float64[3] current_paras
---
float64[3] next_paras
